docs/prompting/thought_generation/chain_of_thought_few_shot/active_prompt.md

```diff
@@ -1,7 +1,102 @@
 ---
-title: ""
-description: ""
-keywords: ""
+title: "Prioritize annotating uncertain examples"
+description: "Active prompting is a method used to identify the most effective examples for human annotation. "
 ---
 
-[wip]
+When we have a large pool of unlabeled examples that could be used in a prompt, how should we decide which examples to manually label?
+
+Active prompting is a method used to identify the most effective examples for human annotation. The process involves four key steps:
+
+1. **Uncertainty Estimation**: Assess the uncertainty of the LLM's predictions on each possible example
+2. **Selection**: Choose the most uncertain examples for human annotation
+3. **Annotation**: Have humans label the selected examples
+4. **Inference**: Use the newly labeled data to improve the LLM's performance
+
+## Uncertainty Estimation
+
+In this step, we define an unsupervised method to measure the uncertainty of an LLM in answering a given example.
+
+!!! example "Uncertainty Estimation Example"
+
+    Let's say we ask an LLM the following query:
+    >query = "Classify the sentiment of this sentence as positive or negative: I am very excited today."
+
+    and the LLM returns:
+    >response = "positive"
+
+    The goal of uncertainty estimation is to answer: **How sure is the LLM in this response?**
+
+In order to do this, we query the LLM with the same example _k_ times. Then, we use the _k_ responses to determine how dissimmilar these responses are. Three possible metrics<sup><a href="https://arxiv.org/abs/2302.12246">1</a></sup> are:
+
+1. Disagreement
```

Comment:
Would be useful to give a short description of what these metrics represent ( single sentence description )